app/src/main/java/com/example/moodtrackerproject/ui/mood/test_results/TestResultsFragment.kt

```diff
@@ -18,10 +20,22 @@ class TestResultsFragment : BaseFragment<TestResultsViewModel, FragmentTestResul
 
     override fun render(props: TestResultsProps) {
         binding?.run {
-            stressBar.max = props.sumStressPoints
-            stressBar.progress = props.stressPoints
-            resultNum.text = "${props.resultPer}%"
-            backButt.click(props.openMood)
+            if (props.testType == 0) {
+                val pers = (DataBaseRepository.stressPoints * 100) / 25
```

Comment:
Add `stressPoints ` to props and try to avoid reference to DataBaseRepository in UI (even it is just a field)